---
title: Food Microbial Ecology lab 
subtitle: INRAE | Micalis institute | Universit√© Paris Saclay
layout: page
callouts: home_callouts
hero_image: /img/FME_Banner_website_V3.png
hero_darken: true
show_sidebar: true
---



# Understanding the microbial ecology of the food chain: towards more sustainable and healthy food.

The FME lab, a research group at MICALIS institute, focuses its research on understanding the mechanisms by which bacteria and fungi interact with food products during spoilage andfermentation . In particular, our team is trying to understand how food microbial diversity may be leveraged to design microbial solutions for more sustainable and healthy fermented food production.  We develop fundamental analysis of food microbial ecology, including metagenomics, metatranscriptomics and metabolomics, comparative genomics and global analyses of the metabolic functions carried out by the different microbial species. As shown in the figure, this scientific strategy is based on four research axes.  A strongly inter-disciplinary strategy combines results and approaches from microbial ecology, high throughput omics data analysis, and computational biology. Our scientific activity creates a strong foreground to develop downstream innovative programs with industrial stakeholders in fermented food areas.

![](https://www.micalis.fr/var/internet6_national_micalis/storage/images/media/images/figure_proet_fme_v12/93444-1-eng-GB/Figure_Proet_FME_V1.png)

## 1. Holistic view of food microbiomes in the food chain.

Our research uses two approaches. The first of these consists of large and ambitious sampling surveys to collect from hundreds to thousands of data points on given industrial environment or type of food. Metagenetic analysis (for example with 16S rRNA genes, ITS or *gyrB* markers) and multivariate discriminant analysis are then performed to unravel how the food production parameters shape the microbial communities. Our work has demonstrated the potential of this type of data (obtained in collaboration with industrial partners) for the development of predictive tools and in characterization of the processing steps that can influence the microbial composition and activity in the context of food production and storage. . The second approach has as its basis a thorough functional analysis of food microbiomes combined using shotgun metagenomics and comparative genomic and functional analysis of new strains and species isolated from food. In this context, we are setting up an easy-access metagenomic study tool integrating public genomic data (genes, functions ...) with metagenomic data (gene profiles and nucleotide variations inferred from shotgun sequencing) and ecological and technological data sets.

## 2. Improving functional analysis of food microbial metabolic interactions.

The metabolic interactions between microbial species are characterized by links between dominant and sub-dominant species or successive populations. The populations may be spatially structured and may also be influenced by environmental factors. The simple mapping of populations and of metagenomes is not sufficient to decipher the underlying interactions that operate within microbial communities during their growth in food, Our scientific approach focuses on the development of metatranscriptomic analyses using computational tools and specifically developed databases. Mapping of the transcriptomes to a database containing several hundred metabolic functions provides a detailed measure of the metabolic functions expressed at different levels of the community, from the species to the whole microbial consortium. In this way, we are able to interpret the interactions taking place at different scales. Our work also involves using non-targeted metabolomics data and integrating these multivariate data using statistical tools. Our work has thus far focused on studying the interactions associated with the spoilage of meat and seafood products and how the reduction of chemical preservatives (salt, nitrite, lactate) could influence these interactions.

## 3. Towards synthetic food microbial ecology.
The object of this axis to develop new microbial consortia to address health or sustainability issues by taking advantage of knowledge gained on microbial interactions and strains genomic sequences. To this end we are developing strategies which will allow us to verify the observations made "*in alimentariis*" and the hypotheses built from these data on metabolic interactions. This involves the design of simplified food systems and the development of tools for modelling the behaviour of artificial microbial consortia. Our "synthetic ecology approach" aims to elaborate hypotheses on the possible metabolic interactions within a microbial community, and to build an experimental approach whose objective is to test them. It is, therefore, a repetitive, iterative strategy whose results are used to reformulate and refine the hypotheses which are then retested experimentally, the aim being always to improve the level of understanding of the interactions when compared to data obtained from real-life food ecosystems.

## 4. Food-Gut interactions.
Food microbes can have major positive, or negative, impacts on the production of various fermented foods, and in the human gut and digestive health. Although a broad diversity of microbial communities can be identified across many fermented foods, significant information about their contribution to human health is lacking. We aim to understand how diversity of dietary microbes, contribute to a healthy gut microbiome and how the long-term consumption of fermented food contributes to a beneficial health impact. Our objective is to use food and gut microbiome data to allow a more relevant health-orientated design of microbial consortia for fermented food and to identify biomarkers related to the health effect of these fermented foods.

